Products/CMFPlone/controlpanel/browser/maintenance.py

```diff
@@ -58,10 +71,17 @@ def handle_shutdown_action(self, action):
                 default=u'You are not allowed to manage the Zope server.'
             )
             return
-        context = aq_inner(self.context)
-        cpanel = context.unrestrictedTraverse('/Control_Panel')
-        result = cpanel.manage_shutdown()
-        return result
+        try:
+            user = '"%s"' % getSecurityManager().getUser().getUserName()
+        except:
```

Comment:
No bare excepts please.
I guess the accepted 'exception' to this rule is that you can catch ConflictError and KeyboardInterrupt and raise them, and do a bare except after this. But I would expect catching ValueError and AttributeError is enough here.